config/config_test.go

```diff
@@ -439,6 +439,12 @@ func TestConfig_Validate(t *testing.T) {
 	*validMultiTask.TerraformProviders = append(*validMultiTask.TerraformProviders,
 		&TerraformProviderConfig{"Y": map[string]interface{}{}})
 
+	validNilTasks := longConfig.Copy()
+	*validMultiTask.Tasks = nil
```

Comment:
I don't quite follow `*validMultiTask.Tasks = nil`. Maybe should "validMultiTask" be "validNilTasks"?

Reply:
yes you're right, copy paste error